wiki/docs/toolchain.md

```diff
@@ -1,17 +1,47 @@
 Toolchain
 ---
 
-To build Mimiker you will need a toolchain, i.e. compiler, linker, ELF tools and
-debugger. The default option is to choose quite recent _LLVM toolchain_ (check
-version in [tools.mk][6]), i.e. `clang`, `lld`, `llvm` from [apt.llvm.org][7],
-and [gdb-multiarch][8]. However you'll need to reconfigure [launch][9] script to
-use same `gdb-multiarch` binary for each target architecture, look for `gdb`
-section and find `binary` key. 
+To build and run Mimiker you will need a toolchain, i.e. compiler, linker, ELF
+tools emulator and debugger.
 
-You need to install patched version of [QEmu][11] as well. Our version solves
-several issues not patched in the mainstream version – please refer to our list
-of [patches][10]. Prebuild package for Debian x86-64 can be found [here][5].
+## Building tools
+The default option is to choose quite recent _LLVM toolchain_ (check version in
+[tools.mk][6]), i.e. `clang`, `lld`, `llvm` from [apt.llvm.org][7].
 
+List of building tools:
+- make
+- clang
+- llvm
+- lld
+- gperf
+- byacc
+- cpio
+
+## Running tools
+Mimiker is run using [QEmu][11] emulator and can be debugged using
+[gdb-multiarch][8]. There is prepared [launch][9] script to automate running the
+Mimiker (it uses tmux to create windows attached to output and debugger).
+
+NOTE:
+If you plan to run Mimiker on MIPS and RISC-V you need to install patched
+version of QEmu as well. Our version solves several issues not patched in the
+mainstream version – please refer to our list of [patches][10]. Prebuild package
+for Debian x86-64 can be found [here][5].
+
+NOTE:
+To use gdb-multiarch you need to reconfigure [launch][9] script to use same
+`gdb-multiarch` binary for each target architecture, look for `gdb` section and
+find `binary` key.
+
+List of running tools:
+- tmux
+- socat
+- gdb-multiarch
+- qemu-system-aarch64
+- qemu patched for our use for MIPS and RISC-V
```

Comment:
you also need python with pexpect, texttable, libtmux libs for all builds. risc-v and arm64 builds require dtc (device-tree-compiler) for preparing blobs

Reply:
Python requirements are in file `requirements.txt` in main directory.

Reply:
Here you have written "running tools". My question is "what is a purpose of listing some tools required for run but not all of them?". I can imagine that somebody want to run Mimiker. They will install packages listed **here** and nothing more and then they got obscure python error that something is missing or even worse - they will got empty tmux window without any log.
From my experience if you start with writing that kind of lists you need to include **everything**.